lib/LoginStrategy.ts

```diff
@@ -1,15 +1,20 @@
 import { BaseStrategy, StrategyOptions } from './BaseStrategy'
 import { Request } from 'express'
 import * as jwt from 'jsonwebtoken'
-import { providers } from 'ethers'
+import { providers, Signer, Wallet, utils } from 'ethers'
 import axios, { AxiosInstance } from 'axios'
 import {
   ethrReg,
   Resolver,
   VoltaAddress1056,
 } from '@ew-did-registry/did-ethr-resolver'
 
-import { lookup, namehash, verifyClaim } from './utils'
+import {
+  createLoginTokenHeadersAndPayload,
```

Comment:
I think `utils` shouldn't really be the place for this function as, IMHO, this starts to move `utils` to be a bit of a function dumping ground. An idea for a refactor is that a `CacheServerClient` class is extracted. This would essentially be a wrapper around an axios client but could have methods like `loginToCacheServer` and `createLoginTokenHeadersAndPayload`. This allows a separation and a grouping of CacheServer functionality.

Reply:
Yeah, I was thinking about the same approach, let's do that 👍

Reply:
I've also added retry option when the response code is 401